main/acle.md

```diff
@@ -2700,7 +2705,7 @@ operations [[G.191]](#G.191):
 This example assumes the implementation preserves the Q flag on return
 from an inline function.
 
-## SIMD intrinsics for 32-bit Operations
+## <span id="32-bit-simd-operations">32-bit SIMD Operations</span>
```

Comment:
Does using `<span>` work after pdf conversion?

Reply:
Nope, see my previous comment in https://github.com/ARM-software/acle/pull/153#discussion_r778448755